VWOSdk/Core/VWOClient.cs

```diff
@@ -676,6 +744,58 @@ public bool FlushEvents()
             LogInfoMessage.NoVariationAllocated(file, userId, campaignKey);
             return new UserAllocationInfo();
         }
+        private UserAllocationInfo normalizeAndFindWinningCampaign(List<BucketedCampaign> shortlistedCampaigns, BucketedCampaign calledCampaign,
+            UserStorageMap userStorageMap, Dictionary<string, dynamic> customVariables,
+           string campaignKey, string apiName, Dictionary<string, dynamic> variationTargetingVariables, bool initIntegration, string userId, string groupName, string groupId)
+        {
+            int currentAllocation = 0;
+            foreach (BucketedCampaign campaign in shortlistedCampaigns)
+            {
+                campaign.Weight = 100.0 / shortlistedCampaigns.Count;
+                int stepFactor = CampaignHelper.GetCampaignBucketingRange(campaign.Weight);
+                if (stepFactor != 0)
+                {
+                    campaign.StartRange = currentAllocation + 1;
+                    campaign.EndRange = currentAllocation + stepFactor;
+
+                    currentAllocation += stepFactor;
+                }
+            }
+            double bucketValue = CampaignAllocator.GetUserHashForCampaign(userId, Convert.ToInt32(groupId));
+            BucketedCampaign winnerCampaign = CampaignHelper.getAllocatedItem(shortlistedCampaigns, bucketValue);
+            if (winnerCampaign != null && winnerCampaign.Id.Equals(calledCampaign.Id))

```

Comment:
missing log called campaign not winner, when winner campaign is not called campaign